off-chain/src/Network/Leader.purs

```diff
@@ -86,13 +87,17 @@ includeAction
   -> UserAction a
   -> Aff (Either IncludeActionError UUID)
 includeAction leaderNode action = do
-  let receivedQueue = getFromLeaderState leaderNode _.receivedActionsRequests
-  queueResponse <- liftEffect $ Ref.new (Right Nothing)
-  _ <- liftEffect $ Queue.put receivedQueue (Right (action /\ queueResponse))
-  value <- loop queueResponse
-  case value of
-    Nothing -> (Left <<< RejectedServerBussy) <$> leaderStateInfo leaderNode
-    (Just uuid) -> pure $ Right uuid
+  inProcess <- isAnotherActionInProcess leaderNode action
+  if inProcess then (Left <<< RejectedServerBussy) <$> leaderStateInfo
```

Comment:
Maybe we need to use a new type for this error? 

I begin to think that using the same types on both sides has begun to add unneeded complexity (like, the user should have its own error Status instead of sharing them with the Server).

Reply:
Will add it to the list of future development possibilities.